packages/sqflite/tizen/src/database_manager.h

```diff
@@ -0,0 +1,66 @@
+#ifndef SQFLITE_DATABASE_MANAGER_H_
+#define SQFLITE_DATABASE_MANAGER_H_
+
+#include <flutter/standard_method_codec.h>
+#include <sqlite3.h>
+
+#include <list>
+#include <string>
+
+namespace sqflite_database {
+typedef std::variant<int64_t, std::string, double, std::vector<uint8_t>,
+                     std::nullptr_t>
+    ResultValue;
+typedef std::vector<ResultValue> Result;
+typedef std::vector<Result> Resultset;
+typedef std::vector<std::string> Columns;
+typedef flutter::EncodableList SQLParameters;
+
+class DatabaseManager {
+ public:
+  static const int kBusyTimeoutMs = 2500;
+
+  DatabaseManager(std::string path, int id, bool single_instance, int log_level)
+      : path_(path),
+        database_id_(id),
+        single_instance_(single_instance),
+        log_level_(log_level){};
+  virtual ~DatabaseManager();
+
+  inline const std::string path() { return path_; };
+  inline const int logLevel() { return log_level_; };
+  inline const bool singleInstance() { return single_instance_; };
```

Comment:
https://google.github.io/styleguide/cppguide.html#Function_Names
```suggestion
  inline const int log_level() { return log_level_; };
  inline const bool single_instance() { return single_instance_; };
```